typings/react/react.d.ts

```diff
@@ -150,6 +150,7 @@ declare namespace __React {
     var DOM: ReactDOM;
     var PropTypes: ReactPropTypes;
```

Comment:
I don't remember changing this either. I'll investigate where they both came from.